R/carehomes.R

```diff
@@ -46,6 +46,10 @@ NULL
 ##' @param prop_noncovid_sympt Proportion of population who do not have
 ##'   covid but have covid-like symptoms
 ##'
+##' @param rel_susc A vector of values representing the relative susceptibility
```

Comment:
I wonder if we should expand this name a little? `rel_susceptibility` perhaps? Or something related to vaccination?